src/pages/react-open-source-contributions.js

```diff
@@ -180,7 +180,15 @@ export const query = graphql`
     }
     gatsbyBlog: file(relativePath: { eq: "open-source/gatsby-blog.png" }) {
       childImageSharp {
-        fixed(width: 66, height: 64) {
+        fixed(width: 66, height: 60) {
```

Comment:
I think this height was changed from 64 to 60 by mistake.

Reply:
Yes, It was by mistake